tools/scripts/move_only_function_specializations.py

```diff
@@ -1,12 +1,11 @@
 # Copyright (c) Microsoft Corporation.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-# This script generates the partial specializations of _Move_only_function_call in <functional>.
+# This script generates the partial specializations of _Function_call in <functional>.
 
-def specialization(cv: str, ref: str, ref_inv: str, noex: str, noex_val: str, callable: str) -> str:
+def specialization(self: str, cv: str, ref: str, ref_inv: str, noex: str, noex_val: str, callable: str) -> str:
     return f"""template <class _Rx, class... _Types>
-class _Move_only_function_call<_Rx(_Types...) {cv} {ref} {noex}>
-    : public _Move_only_function_base<_Rx, {noex_val}, _Types...> {{
+class _Function_call<_Rx(_Types...) {cv} {ref} {noex}> : public _Function_base<_Rx, {noex_val}, _Types...> {{

```

Comment:
Pre-existing: We've had trouble with divergence in the past, so I'd like to add clearer comments about this being generated code.